WordPressKit/Private/WPKitLoggingPrivate.h

```diff
@@ -1,2 +1 @@
-@import CocoaLumberjack;
-extern DDLogLevel ddLogLevel;
+
```

Comment:
Do we need to keep this empty file?

Reply:
I think we can safely remote it. https://github.com/wordpress-mobile/WordPressKit-iOS/pull/551.